crates/turbopack-ecmascript/src/parse.rs

```diff
@@ -303,7 +303,7 @@ async fn parse_content(
                 );
 
                 let mut parser = Parser::new_from(lexer);
-                let program_result = parser.parse_program();
+                let program_result = parser.parse_module().map(Program::Module);
```

Comment:
Parse it as program and convert it to a script instead.

Why? CommonJs modules might contain `with()` statements which can only be parsed as script and not as module

Reply:
the purpose of this change is let next-swc's visitor runs for all script treated one by parse_as_program. If we parse as script, that'll still fail.

so in next-swc we definitely parses everything as module -> let visitor runs, if turbopack might need different internals we need to update offending visitors instead.